internal/cmd/clickhouse/client.go

```diff
@@ -0,0 +1,41 @@
+package streaming
+
+import (
+	"crypto/tls"
+	"fmt"
+	"time"
+
+	"github.com/ClickHouse/clickhouse-go/v2"
+
+	"github.com/openmeterio/openmeter/config"
+)
+
+func GetClient(config config.Configuration) (clickhouse.Conn, error) {
```

Comment:
Not sure if it's a good idea, but this could be a function on the struct behind `config.Aggregation.ClickHouse`.

That approach worked pretty well with the polymorphic configurations (dedupe).

It doesn't have to be the final location though. But at least keeps the input (config) and the constructor together for now.

WDYT?

Reply:
I would love to take that approach. I used this split because we had this discussion with @chrisgacsal when doing similar things in the cloud code-base and it felt strange to him to add these to the config structs. But for simplicity's sake I will take that approach for now.

Reply:
I took the approach to provide the options from the config package. That's aligned with what we do for kafka.